kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -292,6 +302,17 @@
         }
         return { backgroundColor: this.$themeColors.brand.primary.v_900 };
       },
+      oidcProviderFlow() {
+        return global.oidcProviderEnabled && this.nextParam;
+      },
+      nextParam() {
```

Comment:
Django ordinarily has support for this, but I had removed it in our implementation - I actually kind of prefer handling it in the frontend, but we could reinstate the backend functionality if that was preferred.